### PR TITLE
Add default config

### PR DESCRIPTION
**Pull request by TealNerd**
Because it's a pain in the ass to start using humbug without one, so this is actually populated with all the options (except the newbie kit because meh and it's in SAH)

### CODE REVIEW

File: config-default.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+wither_skull_drop_rate: 0.0
+ench_gold_app_edible: false
+ench_book_craftable: false
+cobble_from_lava: true
+cobble_from_lava_scan_radius: 0
+scale_protection_enchant: true
+player_max_health: 20
+fix_rail_dup_bug: true
+tag_on_join: true
+drop_newbie_kit: true
```
Comment (ProgrammerDan) [inline config-default.yml]: Set this to false

File: config-default.yml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+drop_newbie_book: true
```
Comment (ProgrammerDan) [inline config-default.yml]: This too

File: config-default.yml [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+disallow_record_playing: true
```
Comment (ProgrammerDan) [inline config-default.yml]: ::tear::

File: config-default.yml [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+allow_water_in_nether: true
+indestructible_end_portals: true
+drop_ores_as_ore: true
```
Comment (ProgrammerDan) [inline config-default.yml]: Sensible default is probably false.

**Comment by TealNerd** [comment]
lol i wrote a program to automate going through and just generate a config from the defaults programmed in, ill fix it :)

**Comment by TealNerd** [comment]
there should probably also be code in there to auto load this config but w/e

**Comment by redmag3** [comment]
A config is such a good idea, searching for settings in the name of it being a configless "feature" was a neat experiment, but not practical if you hadn't memorized everything.

**Pull request merged**